src/operators/inspect_file.cc

```diff
@@ -62,15 +62,12 @@ bool InspectFile::evaluate(Transaction *transaction, const std::string &str) {
         openstr.append(m_param);
         openstr.append(" ");
         openstr.append(str);
-        if (!(in = popen(openstr.c_str(), "r"))) {
```

Comment:
Can we elaborate on this change?

Reply:
popen can not be exported to be a wasm module in emscripten. It will cause missing symbol when running inside envoy.